src/views/LoanDetails/LoanDetails.js

```diff
@@ -143,15 +143,23 @@ class LoanDetails extends React.Component {
 
   viewFeeFine() {
     const { feesFines: { total } } = this.state;
-    return (total === 0) ? '-' : (
-      <button
-        style={{ color: '#2b75bb' }}
-        onClick={(e) => this.feefinedetails(e)}
-        type="button"
-      >
-        {parseFloat(total).toFixed(2)}
-      </button>
-    );
+    const { stripes } = this.props;
+
+    if (total === 0) return '-';
+
+    const value = parseFloat(total).toFixed(2);
+
+    return stripes.hasPerm('ui-users.accounts')
+      ? (
+        <button
+          style={{ color: '#2b75bb' }}
```

Comment:
I know, this is not about your changes, but it looks like this is a challenge to support the values like '#2b75bb' in the code

Reply:
Makes sense. Done.